windows/src/developer/TIKE/main/UframeTextEditor.pas

```diff
@@ -56,10 +56,12 @@   TframeTextEditor = class(TTIKEForm, IKMDEditActions, IKMDSearchActions, IKMDTe
     FEditorFormat: TEditorFormat;
     FOnChanged: TNotifyEvent;
     FTextFileFormat: TTextFileFormat;
+    FHasError: Boolean;
```

Comment:
Is this used somewhere?

Reply:
Thanks, good catch. Ended up using `FErrorLine` instead.